src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -339,6 +339,18 @@ public void setup(boolean gameStarting) {
 
     if (gameStarted) {
       adjustSplitter();
+
+      if (gameStarting) {
+        SwingUtilities.invokeLater(new Runnable() {
+          @Override
+          public void run() {
+            Logger logger = GameModule.getGameModule().getLogger();
+            if (logger instanceof BasicLogger) {
+              ((BasicLogger)logger).queryNewLogFile(true);
+            }
+          }
+        });
+      }
```

Comment:
I'm curious, does this actually need to be run via `invokeLater`? The reason the splitter adjustment below this does is that it is Totally Broken, and I wasn't expecting showing the log file dialog to be Totally Broken.

Reply:
Or do you just want it to happen _after_ the splitter is adjusted?

Reply:
It doesn't care about the splitter, it cares about all the other wonky game components already having been initialized

Reply:
One example - StartupGlobalKeyCommands run during an invokeLater, so to have the logging begin after THEM, I definitely need invokeLater.